extensions/amp-story/amp-story-page-attachment.md

```diff
@@ -72,7 +72,19 @@ Values: "light" (default), "dark"
 >
 ```
 
-## Allowed content and components
+## Linked content
+
+When providing a `href` attribute as page attachment, the respective URL is opened when the user activates the page attachment.
```

Comment:
Wdyt about adding one line about the fact that this is a page navigation on the web, but can be handled different in native contexts (in-app browser, customized app experience...)?

Reply:
I don't think we should specify what exactly happens.